sum_of_values' may be used uninitialized in this function

Found with gcc-10 building debug:
```
[ 74%] Building CXX object xtp/src/libxtp/CMakeFiles/votca_xtp.dir/kmccalculator.cc.o
cd /builds/votca/votca/votca/build/xtp/src/libxtp && /usr/lib64/ccache/g++  -DBOOST_UBLAS_NDEBUG -Dvotca_xtp_EXPORTS -I/builds/votca/votca/votca/build/xtp/src/libxtp -I/builds/votca/votca/votca/xtp/include -I/builds/votca/votca/votca/build/xtp/include -I/builds/votca/votca/votca/csg/include -I/builds/votca/votca/votca/tools/include -I/builds/votca/votca/votca/build/tools/include -isystem /usr/include/eigen3  -Wall -Wextra -Wpedantic -Wshadow -Wconversion -Werror -fopenmp -O2 -DNDEBUG -fPIC   -std=c++14 -o CMakeFiles/votca_xtp.dir/kmccalculator.cc.o -c /builds/votca/votca/votca/xtp/src/libxtp/kmccalculator.cc
In file included from /builds/votca/votca/votca/xtp/include/votca/xtp/gnode.h:23,
                 from /builds/votca/votca/votca/xtp/src/libxtp/kmccalculator.cc:21:
/builds/votca/votca/votca/xtp/include/votca/xtp/huffmantree.h: In member function 'void votca::xtp::KMCCalculator::LoadGraph(votca::xtp::Topology&)':
/builds/votca/votca/votca/xtp/include/votca/xtp/huffmantree.h:28:7: error: '*((void*)(&<anonymous>)+104).votca::xtp::huffmanTree<votca::xtp::GLink>::sum_of_values' may be used uninitialized in this function [-Werror=maybe-uninitialized]
   28 | class huffmanTree {
      |       ^~~~~~~~~~~
cc1plus: all warnings being treated as errors
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@ For more detailed information about the changes see the history of the
 * add ENABLE_HIGH_MEMORY_TESTS cmake option (#356)
 * fix copyright (#363)
 * remove old doxygen target (#365)
+* fix some gcc10 warnings (#376)
 
 ## Version 1.6_rc1 (released 04.12.19)
  * completely new statefile in hdf5 format for larger systems

--- a/include/votca/xtp/huffmantree.h
+++ b/include/votca/xtp/huffmantree.h
@@ -223,7 +223,7 @@ class huffmanTree {
 
   std::vector<huffmanNode<T>> htree;
   bool treeIsMade = false;
-  double sum_of_values;
+  double sum_of_values = 0.0;
   std::vector<T> *events = nullptr;
 };
 

